Add `Taiko Convert Tournament v4` wiki article (#8157)
ppy/osu-wiki: wiki/Tournaments/TCT/v4/en.md

| **Foukat cukr do prdele** | **4** | 1 | can we just play this in std | [#1](https://osu.ppy.sh/community/matches/101600161) |
| **B-Team** | **4** | 1 | Universidad de Chile | [#1](https://osu.ppy.sh/community/matches/101601755) |
| **Tiger Mafia** | **4** | 0 | 404 enrin not found | [#1](https://osu.ppy.sh/community/matches/101603190) |
| **Lipton** | **4** | 0 | Dribblers | [#1](https://osu.ppy.sh/community/matches/101603313) |
| **israel** | **4** | 1 | Foukat cukr do prdele | [#1](https://osu.ppy.sh/community/matches/101603194) |
| a dom | 3 | **4** | **SwitzerFrance** | [#1](https://osu.ppy.sh/community/matches/101608010) |
| **not hehe** | **4** | 0 | Hoteddi\_18 | [#1](https://osu.ppy.sh/community/matches/101610813) |
| **Tiger Mafia** | **4** | 0 | Imagine Playing Basketball with Jesus and Getting Crossed Up | [#1](https://osu.ppy.sh/community/matches/101613192) |

### Qualifiers

Detailed statistics for this round can be found [here](https://docs.google.com/spreadsheets/d/1irT-oEpMu6QfUzKBosyeZJtfKZzcqKmRYAJu9nyWj7s/edit) alongside the final standings on which player made it through to the next stage.

## Ruleset

### General rules

1. A valid team formation will be:

>    - A participant with rank range between #1 and #150

Reviewer: ```suggestion
   - A participant with rank between #1 and #150
```

>    - A participant with rank range below #151

Author: Updated, is that how I suppose to word it?

>    - A participant with rank range between #151 or below

Reviewer: ```suggestion
   - A participant with rank of #150 or below
```

6. Even in a tournament scene osu! is still a game. Please respect each other to everyone, be friendly, and fair play.

### In-game conditions

#### Qualifiers

1. Every players will be invited at the time to the room.
2. There will be no warmups, no ban phase, and no pick phase.
3. In the qualifiers round, each map will be played twice, or you can play once and leave if you think you've set your best score.
4. There will be a maximum time of **2 minutes** in between maps. The referee will use `!mp start 10` command to start the game.
5. If a player gets disconnected for some reason in the first 10 seconds of the song, the referee will abort the game and restart it.
   - If it happens after 10 seconds, the score before the disconnect will be counted.
6. The referee can penalise teams if any player present bad behaviour in the game room.
7. Once the match has finished, the room will be completely closed.
8. If you fail, the score **will still be counted**.

#### Group Stage and Knockouts

1. Every players will be invited at the time to the room.
2. The referee can wait a maximum of **10 minutes** for each player to attend.

>    - If the player doesn't come, the referee will ask the other team for a 1v2 start. It will depend on the other team if they want to play or not.

Reviewer: ```suggestion
   - If a player does not arrive, the referee will ask the other team for a 1v2 start. It will depend on the other team if they want to play or not.
```

>    - Have in mind if there's a 1v1 match you are forced to play unless you want a forfeit.

Reviewer: ```suggestion
   - Keep in mind that abstaining from a 1v1 is equivalent to forfeiting the match.
```

> 3. Captains will decide if they want to pick a warmup map (maximum limit is **3:30**) and order doesn't matter.

Reviewer: ```suggestion
3. Captains will decide if they want to pick a warmup map (maximum limit is **3:30**) and the order does not matter.
```

> 4. Captains will use `!roll` command. Lower roll will start banning then higher roll bans (1 ban each team).

Reviewer: ```suggestion
4. Captains will use the `!roll` command. Each team will ban 1 map starting from the one who rolled the lowest.
```

> 5. After banning stage is ready, higher roll picks their first map then lower roll picks theirs. Continue until the match is done.

Reviewer: ```suggestion
5. After the bans, the team that got the higher roll picks their first map then the other team picks theirs. This is continued until the match is done.
```

6. There will be a **2 minutes** time limit for all phases.

> 7. If the time limit is exceeded

Reviewer: ```suggestion
7. If the time limit is exceeded:
```

>    - Ban: The ban is treated as none.

Reviewer: ```suggestion
   - Ban: No beatmap will be banned.
```

>    - Pick: The right is transferred to your opponent. It does not affect the subsequent order.

Reviewer: ```suggestion
   - Pick: The pick turn is transferred to the opponent. This does not affect the subsequent order.
```

   - Before the start: The referee will use `!mp start 10` to start the game.
8. If a player gets disconnected for some reason in the first 10 seconds of the song, the referee will abort the game and restart it.
   - If it happens after 10 seconds, the score before the disconnect will be counted.

> 9. The referee can penalise teams if any player present bad behaviour in the game room.

Reviewer: ```suggestion
9. The referee can penalise teams if any player presents bad behaviour in the game room.
```

> 10. Once the match has finished, the room will be completely closed.

Reviewer: ```suggestion
10. Once the match has finished, the room will be closed.
```

> 11. FreeMod maps are force mod so at least 1 team member **has to pick a mod**.

Reviewer: ```suggestion
11. FreeMod maps are *force mod*, so at least 1 team member **has to pick a mod**.
```

> 12. Team members aren't allowed to pick the same mod combination.

Reviewer: ```suggestion
12. Team members are not allowed to pick the same mod combination.
```

> 13. If a player fails in a song, and doesn't revive then his score is not counted. In the case he revives before map's end it counts. if every player fails, then the referee is going to make a sum of their scores then comparing.

Reviewer: ```suggestion
13. If a player fails in a song and does not revive, then their score is not counted, but if they do revive it will count. If every player in a team fails, then the sum of their scores will count.
```

> 14. If there is a tie, then there will be no point added to any team, map will be considered as played. If there is a tie on the tiebreaker, then teams are going to repeat it with FreeMod conditions.

Reviewer: ```suggestion
14. If there is a tie, then no points are added to any team, but the map will be considered as played. If there is a tie on the tiebreaker, then then a rematch will happen using FreeMod conditions.
```

> ### Winning conditions

Reviewer: ```suggestion
### Win conditions
```
